### PR TITLE
docs: improved the syntax in the codebase best practice docs

### LINKED ISSUE #48884: improve the syntax in the codebase best practice docs
**Describe the feature**
In these strings, we are using `- -` to indicate these are hint/solution, this cased confusion in translation, here is the issue https://freecodecamp.crowdin.com/translate/457d487b1586f10d2fba3c4ccb9c5453/26540/en-it?filter=basic&value=0#q=1233284.

> - - Use Flexbox and Grid layouts instead, as they have RTL support already built-in, and those will be easier to maintain and review.

> - - Use logical properties for them: You can add the same spacing by using `padding-inline-end` and `margin-inline-start`, and you won't need to worry about RTL layout, as they follow where the line start and ends, and you won't need to add any extra values in the RTL files, so people won't need to remember to change the same values in two files.

Here are the strings:
https://github.com/freeCodeCamp/freeCodeCamp/blob/ef1c90ecd7ed923b9ec82ac52730951cdd7147c3/docs/codebase-best-practices.md#L18-L20?plain=1

Dario recommended a new format and I kinda like it. Here how they should look like: 

> - → Use Flexbox and Grid layouts instead, as they have RTL support already built-in, and those will be easier to maintain and review.

> - → Use logical properties for them: You can add the same spacing by using `padding-inline-end` and `margin-inline-start`, and you won't need to worry about RTL layout, as they follow where the line start and ends, and you won't need to add any extra values in the RTL files, so people won't need to remember to change the same values in two files.

---

If you would like to fix this issue, please make sure you read [our guidelines for contributing](https://contribute.freecodecamp.org/#/), we prioritize contributors following the instructions in our guides. Join us in [our chat room](https://discord.gg/PRyKn3Vbay) or [the forum](https://forum.freecodecamp.org/c/contributors/3) if you need help contributing, our moderators will guide you through this.

### PR DESCRIPTION
**Pull request by pratikbhangire123**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #48884

### CODE REVIEW

**Comment by gikf** [comment]
As a new contributor, we encourage you to read our [contributing guidelines](https://contribute.freecodecamp.org).

We expect our contributors to be aware of the process specific to this project. Following the guidelines religiously earns you the respect of fellow maintainers and saves everyone time.

Some examples of this are:

1. Do not edit files directly through GitHub – while you can, it's not a good idea.
2. Make sure you follow the PR checklist and not just tick things off; otherwise, we won't take you seriously.
3. Use the correct way to link issues in the description of the PR by updating the `XXXXXX`. Do not just add issue numbers everywhere and anywhere you feel like.
4. Keep mentions and review requests to a minimum. We understand you are excited about contributing, and our maintainers will get back to you as soon as they get a chance.
5. Do not work directly off your `main` branch - create a new branch for the changes you are working on.

We appreciate you taking the time to help us, and we hope to see more contributions from you.

Happy Contributing.

**Comment by pratikbhangire123** [comment]
Hey @gikf, sorry if I have made any mistake but,

1. I have edited this doc from my local machine.
2. Worked on it from a different branch.
3. Added the issue number where it is mentioned to add.

Can you tell me what exactly I did wrong?

BTW, Thanks for your comment, I'm just a beginner at contributing :)

**Comment by gikf** [comment]
Hey @pratikbhangire123

thanks for taking another look at the checklist. Could you also change the PR title to follow more closely the convention described in https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title?

File: docs/codebase-best-practices.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ The `!important` keyword may be used to override values in some cases (for ex.:
 
 ### RTL support
 
-We are striving to support right-to-left (RTL) layout in the codebase for languages that are read in this direction. For this you need be mindful of how to style components. Here are a quick role of thumbs to follow:
+We are striving to support right-to-left (RTL) layout in the codebase for languages that are read in this direction. For this you need be mindful of how to style components. Here are some quick rules of thumb to follow:
```
Comment (gikf) [inline docs/codebase-best-practices.md:15]: This seems to be change unrelated to the linked issue, please either remove commit making the change from PR, or revert the text changes.

```suggestion
We are striving to support right-to-left (RTL) layout in the codebase for languages that are read in this direction. For this you need be mindful of how to style components. Here are a quick role of thumbs to follow:
```

Comment (pratikbhangire123) [reply docs/codebase-best-practices.md]: Reverted the text changes and renamed the PR, please check it out!

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @pratikbhangire123, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Pull request merged**